schemas/imaging/acquisition.json

```diff
@@ -49,6 +49,95 @@
 			"format": "date-time",
 			"title": "Session end time"
 		},
+		"local_storage_directory": {
```

Comment:
I'm not opposed to keeping these directories here, but we will be moving data around and it will be hard to keep this straight.